### ISSUE TITLE
[morethantv] Exception (morethantv): Request to https://www.morethan.tv/torrents.php?searchstr=

### ISSUE THREAD

**Issue by cdonovan** [Jackett/Jackett#2268]
**Please use the search bar** at the top of the page and make sure you are not creating an already submitted issue. Duplicating issues makes it more difficult for everyone to follow. Your issue may have already been solved in the past as well.

Provide a description of the feature request or bug, the more details the better.
If you are experiencing an issue with a tracker, a **full log must be included**. Instructions for obtaining logs are here: https://github.com/Jackett/Jackett#troubleshooting

Jackett version:  0.8.500.0
Mono version (if not using Windows):  5.4.1.6

-------------------------------
Getting errors fetching from morethan.tv.  The error in the logs is:
On attempt 1 checking for results from MoreThanTV: Request to https://www.morethan.tv/torrents.php?searchstr=&tags_type=1&order_by=time&order_way=desc&group_results=1&action=basic&searchsubmit=1 failed (Error BadGateway) - The tracker seems to be down.

I went to the URL in the error manually and didn't receive the 502 (bad gateway) error that Jackett is receiving.  Let me know if there's ay more info needed to help debug this.

**Comment by kaso17** [comment]
Works fine for me, is this still a problem? looks like a temporary instability issue to me.

**Comment by cdonovan** [comment]
It was still doing it earlier this morning for me (from yesterday around 3pm PST.  However, I was able to get it running again just now by stopping the service, wiping the Jackett config folder, restarting and reconfiguring the trackers.  Previously, I tried just deleting and re-adding the mtv indexer itself within the UI, but it was still using an old (cached?) session.  In any event, it is resolved now.  Thanks for the quick follow-up!  Should this issue creep back again, I'll be sure to get some detailed logs.
